riak_test/yokozuna_essential.erl

```diff
@@ -79,6 +79,7 @@ confirm() ->
             verify_deletes(Cluster2, ?INDEX, ?NUM_KEYS, KeysDeleted),
             ok = test_escaped_key(Cluster2),
             verify_unique_id(Cluster2, PBConns),
+            %% verify_delete_index(Cluster2, PBConns),
```

Comment:
Do we want this to be commented? If so, it would be nice to also have a comment explaining when/why someone might want to uncomment this part of the test. If not, perhaps we should uncomment it :).